src/System.Text.Json/pkg/Microsoft.Bcl.Json.Sources.pkgproj

```diff
@@ -14,7 +14,10 @@
        the Common SR.cs into SourcePackageFiles. -->
   <Target Name="GetSourcesToPackage" BeforeTargets="ExpandProjectReferences">
     <ItemGroup>
-      <_ProjectsToBuild Include="../src/System.Text.Json.csproj" UndefineProperties="Configuration" AdditionalProperties="TargetGroup=netstandard"/>
+      <!-- Ensure that we undefine BuildConfiguration and set it to netstandard for getting the right sources to package. -->
+      <_ProjectsToBuild Include="../src/System.Text.Json.csproj" 
+                        UndefineProperties="Configuration;BuildConfiguration" 
+                        AdditionalProperties="TargetGroup=netstandard;BuildConfiguration=netstandard" />
```

Comment:
So, we need both TargetGroup and BuildConfiguration?

Reply:
TargetGroup is for direct builds (`dotnet msbuild Microsoft.Bcl.Json.Sources.pkgproj`) whereas BuildConfiguration is for building from the root which evaluates the configuration that gets picked beforehand.

Reply:
> TargetGroup is for direct builds

It doesn't seem necessary to me, but its fine to keep it if you think its useful.